nix/gecko_env.nix

```
@@ -5,8 +5,8 @@ let
   inherit (releng_pkgs.pkgs) rustChannelOf bash autoconf213 clang_7 llvm_7 llvmPackages_7 gcc-unwrapped glibc fetchFromGitHub unzip zip openjdk python2Packages sqlite zlib nasm;
   inherit (releng_pkgs.pkgs.devEnv) gecko;
 
-  # Rust 1.31.1
-  rustChannel' = rustChannelOf { date = "2018-12-20"; channel = "stable"; };
+  # Rust 1.32.0
+  rustChannel' = rustChannelOf { date = "2019-01-17"; channel = "stable"; };
```

```suggestion
  rustChannel' = rustChannelOf { date = "2019-01-18"; channel = "stable"; };
```

https://en.wikipedia.org/wiki/File:Alitalia-17.jpg